themes/base.theme.bash

```diff
@@ -359,7 +359,9 @@ function hg_prompt_vars {
 
 	if [ -f "$HG_ROOT/branch" ]; then
```

Comment:
I am not that expert on mercurial, but can a bookmark exist without branch? if so, this checks makes it impossible
You can add another `if else` case with the bookmark, instead of using this `branch_or_bookmark` variable

Reply:
I think there's always a branch, but the more I think about it, the bookmark is more like a git tag.

I will change it to always show the branch, and if there's a bookmark, also display that.